sklearn/utils/_repr_html/estimator.css

```diff
@@ -39,8 +39,7 @@
 
 #$id input.sk-hidden--visually {
   border: 0;
-  clip: rect(1px 1px 1px 1px);
-  clip: rect(1px, 1px, 1px, 1px);
+  clip-path: inset(100%);
```

Comment:
css property `clip` is deprecated https://developer.mozilla.org/en-US/docs/Web/CSS/clip